packages/spatial/src/renderer/components/VisibleComponent.ts

```diff
@@ -28,7 +28,17 @@ import { Entity } from '@ir-engine/ecs/src/Entity'
 
 export const VisibleComponent = defineComponent({
   name: 'VisibleComponent',
-  jsonID: 'EE_visible'
+  jsonID: 'EE_visible',
+
+  onSet(entity, component, json) {
```

Comment:
i think the onSet here is redundant, we only need a toJSON

Reply:
in fact, i think the visible component should return `{}` instead of `{ value: true }`, same as what other gltf extensions wih no properties do https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_materials_unlit

Reply:
we should make `serializeComponent` return `{}` if there is no schema on the component